app/common/verifier/verifier.js

```diff
@@ -0,0 +1,256 @@
+// import log       from 'fancy-log';
+import isUrl from 'is-url';
+import fs from 'fs';
+import fetch from 'node-fetch';
+import * as forms from '../../../mocked_backend/api/v3/forms.json';
+import * as tags from '../../../mocked_backend/api/v3/tags.json';
+import * as features from '../../../mocked_backend/api/v3/config/features.json';
+import * as map from '../../../mocked_backend/api/v3/config/map.json';
+
+const verifyStatus = function (url, options) {
+    let results;
+    return fetch(url, options)
+        .then(response => {
+            switch (response.status.toString()) {
+                case '200':
+                    results = {
+                        type: 'confirmation',
+                        messages: ['All is good. This is the expected result.'],
+                        url: response.url,
+                        status: response.status
+                    };
+                    break;
+                case '404':
+                    results = {
+                        type: 'error',
+                        messages: ['Make sure the API\'s BACKEND_URL in the .env file is the base URL to your Platform API.'],
+                        url: response.url,
+                        status: response.status
+                    };
+                    break;
+                case '403':
+                    results = {
+                        type: 'error',
+                        messages: ['Make sure the API\'s BACKEND_URL in the .env file is the base URL to your Platform API.'],
+                        url: response.url,
+                        status: response.status
+                    };
+                    break;
+                case '500':
+                    results = {
+                        type: 'error',
+                        messages: ['Oh noes. This does not look good.', 'Please check storage/logs in the Platform API, and see what the logs say about this error.'],
+                        url: response.url,
+                        status: response.status
+                    };
+                    break;
+            }
+            return results;
+        })
+        .catch(error => {
+            results = {
+                    type: 'error',
+                    messages: ['The server could not be reached or there was an error in the request', 'Make sure the BACKEND_URL is in your .env-file', error],
+                    url: url
+                };
+            return results;
+        });
+};
+
+const verifyEnv = function (env) {
+    let messages = [];
+    try {
+        // fs is not working in the browser, how handle it?
+        fs.accessSync('.env');
```

Comment:
fs is not working in the browser, but waiting for us to solve the main-problem with where to put the browser-verifier before finding a solution for it.